examples/fabric/nested_ingestion_pipeline/README.md

```diff
@@ -0,0 +1,96 @@
+# Nested pipelines
+
+The following example, exemplifies the utilization of nested pipelines including control activities like ForEach and If.
+
+![Nested Pipelines](pl_main_pipeline.png)
+
+The main pipeline is composed of 2 top activities:
+
+- a lookup activity named "Read Configuration File
+- a ForEach activity names "ForEach"
+
+Within the ForEach activity, there is a nested If activity called "If New Or Updated" and if this activity evaluates to True, a Invoke Pipeline activity named "Invoke Ingestion Pipeline" is triggered.
+
+The Ingestion pipeline is composed also by two activities:
+
+- a Copy activity named "Copy NYCData to ADLS" that copies from the Web to an ADLS Gen2 storage,
+- a Copy activity named "CopyNYCData to Lakehouse" that copies from the ADLS Gen2 storage to the Lakehouse.
```

Comment:
```suggestion
- a Copy activity named "Copy NYCData to Lakehouse" that copies from the ADLS Gen2 storage to the Lakehouse.
```

Reply:
activity already contains the space

Reply:
I've renamed this ones based on the feedback from the previous comment. Now they are called "Copy NYCData from Web to ADLS" and "Copy NYCData from ADLS to Lakehouse" and corrected the typo.